apiserver/routers.go

```diff
@@ -63,8 +63,18 @@ func recoveryHandler() mux.MiddlewareFunc {
 	return handlers.RecoveryHandler(handlers.PrintRecoveryStack(true))
 }
 
-func timeoutHandler() mux.MiddlewareFunc {
+func timeoutHandler(timeout time.Duration) mux.MiddlewareFunc {
 	return func(h http.Handler) http.Handler {
-		return http.TimeoutHandler(h, time.Second*30, "")
+		return http.TimeoutHandler(h, timeout, "")
+	}
+}
+
+func requestContextInjector(timeout time.Duration) mux.MiddlewareFunc {
```

Comment:
didn't understand how this is being used.

Reply:
ah will remove. timeout handler accepts a context from the request itself, which I was using this to override with a per-request timeout. I was going to use this in UT to force timeouts without mocking a clock (since timeouthandler doesn't accept one).

instead I moved the timeout to the server level, and configured it to zero for UT.